docs/source/hints.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Composition hints
```

Comment:
I've decided to keep this name instead of "Composition linting" (that doesn't seem to accurately describe it) or "Composition rules", which I feel could be confused with the [breaking type of composition rules](https://www.apollographql.com/docs/federation/federated-types/composition/#rules-of-composition).

Open to suggestions!